assets/javascript/tm.js

```diff
@@ -0,0 +1,27 @@
+// Ticketmaster API
+// use to retrieve event information
+var tmApiKey = "8F1kGz3ndgSx2auYBTZt2UCp5S3v8b5G";
+var tmEventId = "vv178Z4JGkS6JSZS"
```

Comment:
Should this end is a semi colon or the other var not have a semi colon. It doesn't seem to  impact code functionality, I was just asking

Reply:
Technically it "should" terminate with a semicolon.  JS is not strict about this.  I will fix for consistency and "correctness" in next commit.